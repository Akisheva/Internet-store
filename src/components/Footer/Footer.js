import React from 'react';

import "./footer.scss"

const Footer = () => {

    return(
        <footer className='contacts'>
            <div className='contacts-wrapper'>
                <p>Copyright &copy; 2022 Akisheva</p>
                <p>phone 0977777711</p>
                <p>Working hours: 9am - 6pm</p>
            </div>
        </footer>
        )
}
export default Footer