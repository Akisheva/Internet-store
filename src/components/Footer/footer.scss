.contacts-wrapper {
    display: flex;
    justify-content: space-around;
    flex: 1 1 auto;
    font-size: 20px;
    background-color: #dfe1e1;
    padding: 30px;
}