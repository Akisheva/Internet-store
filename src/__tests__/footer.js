import renderer from 'react-test-renderer';
import Footer from '../components/Footer'

test('SnapShot', () => {
      const footerComp = renderer.create(<Footer />).toJSON()
      expect(footerComp).toMatchSnapshot();
    });