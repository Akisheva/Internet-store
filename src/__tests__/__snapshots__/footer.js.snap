// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapShot 1`] = `
<footer
  className="contacts"
>
  <div
    className="contacts-wrapper"
  >
    <p>
      Copyright Â© 2022 Akisheva
    </p>
    <p>
      phone 0977777711
    </p>
    <p>
      Working hours: 9am - 6pm
    </p>
  </div>
</footer>
`;
